Serving-patterns/Prediction-circuit-break-pattern/design_ko.md

```diff
@@ -0,0 +1,31 @@
+# Prediction circuit break pattern
+
+## Usecase
+- 예측 요청량 또는 빈도가 갑자기 증가할 가능성이 있는 경우
+- 갑작스러운 요청에 빠르게 응답할만큼 빠르게 스케일 아웃할 수 없는 경우
+- 100% 응답할 필요가 없는 경우
+
+## Architecture
+캠페인이나 외부 이벤트 때문에 웹 서비스나 관련 서비스에 접속이 증가하는 경우가 있습니다. 만약 클라우드 서비스 또는 쿠버네티스를 사용한다면, 증가된 요청량이 스케일 아웃 속도를 넘어서는 경우가 많지만 서비스를 스케일 아웃해 요청에 응답할 수 있습니다. 특히 서버 또는 컨테이너 이미지에 모델 파일이 포함된 경우, 모델을 실행하거나 파일을 다운로드해야 합니다.
```

Comment:
```suggestion
캠페인이나 외부 이벤트 때문에 웹 서비스나 관련 서비스에 접속이 증가하는 경우가 있습니다. 만약 클라우드 서비스 또는 쿠버네티스를 사용한다면 요청량만큼 스케일 아웃을 할 수 있지만, 스케일 아웃 속도보다 요청량이 증가하는 속도가 더 빠른 경우가 많습니다. 특히 서버 또는 컨테이너 이미지에 모델 파일이 포함된 경우, 모델을 실행하거나 다운로드 받는데 시간이 오래 걸릴 수 있습니다. 
```

영문에서  launching or downloading it may take a file. 로 되어있는데 일본어를 보니 왠지 take a while 혹은 take time 을 잘못 쓴것 같아용!